docs/processors/parallel-processors.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Parallel Processors'
+sidebar_position: 3
+---
+
+Conduit allows you to configure a processor to process records in parallel. This
+means that multiple records can be processed simultaneously by one processor,
+which can be particularly beneficial for processors performing time-consuming
+computations on a fast stream of records.
+
+:::tip Ordering guarantee
+
+When processors are configured to process records concurrently, they guarantee
+that the order of records won't change. Although records may be processed out of
+order, the resulting order of records coming out of the processor is guaranteed
+to be unchanged.
+
+:::
+
+For instance, consider a pipeline that needs to process 100 records per second.
+One of the processors in the pipeline needs to retrieve data from a server, and
+this operation typically takes 1 second to complete. Since this operation is
+executed for every record, sequential processing won't keep up with the stream.
+However, by configuring the processor to use 200 parallel workers, we can
```

Comment:
why 200? the parallel 100 workers should finish processing in 1 second too, unless you intentionally wanted a higher number, then I think 200 is a bit confusing for the example, and the user might think that they need double the number of records, wdyt?

Reply:
I can change the example to 100 to make it clearer. I wanted to show that you want to configure a bit more workers to have some room for higher loads. But I can try mentioning this at the end of the example.